aesara/tensor/random/basic.py

```diff
@@ -69,6 +69,34 @@ def rng_fn(cls, *args, **kwargs):
         )
 
 
+class ARMArv(RandomVariable):
+    name = "ARMA"
+    ndim_supp = 1
+    ndims_params = [1, 1]
+    dtype = "floatX"
+    _print_name = ("ARMA", "\\operatorname{ARMA}")
+
+    def __call__(
+            self,
+            loc: np.ndarray,
+            scale: np.ndarray,
+    ) -> RandomVariable:
+        return super().__call__(loc, scale)
+
+    @classmethod
+    def rng_fn_scipy(
```

Comment:
```suggestion
    def rng_fn(
```

The `rng_fn_scipy` method is only used by `ScipyRandomVariable`, and it's simply re-adds dimensions that are mistakenly dropped by some of SciPy's distributions.  `scipy.signal.lfilter` might not drop dimensions, so it's not clear whether or not you should be using `ScipyRandomVariable`.